elements/rh-cta/rh-cta.css

```diff
@@ -187,33 +197,41 @@ svg {
   --rh-cta-hover-color: var(--rh-context-color-link-hover, #004080);
   --rh-cta-hover-text-decoration: none;
   --rh-cta-focus-background-color: transparent;
-  --rh-cta-focus-container-background-color: #0066cc1a; /* --rh-color-border-interactive-on-light with 10% opacity */
+
+  /* --rh-color-border-interactive-on-light with 10% opacity */
+  --rh-cta-focus-container-background-color: #0066cc1a;
   --rh-cta-focus-border-color: transparent;
   --rh-cta-focus-color: var(--rh-context-light-color-focus, #0066cc);
   --rh-cta-focus-inner-border-color: transparent;
   --rh-cta-focus-text-decoration: none;
-  --rh-cta-active-container-background-color: #0066cc1a; /* --rh-color-border-interactive-on-light with 10% opacity */
+
+  /* --rh-color-border-interactive-on-light with 10% opacity */
+  --rh-cta-active-container-background-color: #0066cc1a;
```

Comment:
Out of curiosity, what does the 1a at the end of the hex code mean?

Reply:
That's the alpha channel

https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color

Reply:
we might consider vending hsl tokens as well, which would make this easier to validate in css.